bot.py

```diff
@@ -66,37 +67,22 @@
     },
 }
 
-# initializing empty states
+# starting states
 current_node = "welcome"
 form = {"area": "", "cuisine": "", "price_range": ""}
 suggestions = []
 
 
-# TODO: update with type_match_ls.py logic
 def extract_cuisine(utt):
-    if "chinese" in utt:
-        return "chinese"
-    else:
-        return ""
+    return type_match_ls.extract_food(utt)
 
 
 def extract_area(utt):
-    if "ny" in utt:
-        return "ny"
-    else:
-        return ""
+    return type_match_ls.extract_area(utt)
 
 
 def extract_price_range(utt):
-    if "cheap" in utt:
-        return "cheap"
-    else:
-        return ""
-
-
-# TODO: update with whichever classification model we choose
-def classify(utt):
```

Comment:
I thought it'd be best if we just call classifier(), instead of classify() calling classifier, if that makes sense